openpype/hosts/substancepainter/plugins/load/load_mesh.py

```diff
@@ -0,0 +1,123 @@
+        ),
+        qargparse.Boolean(
+            "import_cameras",
+            default=True,
+            help="Import cameras from the mesh file."
+        )
+    ]
+
+    def load(self, context, name, namespace, data):
+
+        # Get user inputs
+        import_cameras = data.get("import_cameras", True)
+        preserve_strokes = data.get("preserve_strokes", True)
+
+        if not substance_painter.project.is_open():
+            # Allow to 'initialize' a new project
+            result = prompt_new_file_with_mesh(mesh_filepath=self.fname)
+            if not result:
+                self.log.info("User cancelled new project prompt.")
+                return
+
+        else:
+            # Reload the mesh
+            settings = substance_painter.project.MeshReloadingSettings(
+                import_cameras=import_cameras,
+                preserve_strokes=preserve_strokes
+            )
+
+            def on_mesh_reload(status: substance_painter.project.ReloadMeshStatus):  # noqa
+                if status == substance_painter.project.ReloadMeshStatus.SUCCESS:  # noqa
+                    print("Reload succeeded")
+                else:
+                    raise RuntimeError("Reload of mesh failed")
+
+            path = self.fname
+            substance_painter.project.reload_mesh(path,
+                                                  settings,
+                                                  on_mesh_reload)
+
+        # Store container
+        container = {}
+        project_mesh_object_name = "_ProjectMesh_"
+        imprint_container(container,
+                          name=project_mesh_object_name,
+                          namespace=project_mesh_object_name,
+                          context=context,
+                          loader=self)
+
+        # We want store some options for updating to keep consistent behavior
+        # from the user's original choice. We don't store 'preserve_strokes'
+        # as we always preserve strokes on updates.
+        container["options"] = {
+            "import_cameras": import_cameras,
+        }
+
+        set_container_metadata(project_mesh_object_name, container)
+
+    def switch(self, container, representation):
+        self.update(container, representation)
+
+    def update(self, container, representation):
+
+        path = get_representation_path(representation)
+
+        # Reload the mesh
+        container_options = container.get("options", {})
+        settings = substance_painter.project.MeshReloadingSettings(
+            import_cameras=container_options.get("import_cameras", True),
+            preserve_strokes=True
+        )
+
+        def on_mesh_reload(status: substance_painter.project.ReloadMeshStatus):
+            if status == substance_painter.project.ReloadMeshStatus.SUCCESS:
+                print("Reload succeeded")
+            else:
+                raise RuntimeError("Reload of mesh failed")
```

Comment:
```suggestion
                raise LoaderError("Reload of mesh failed")
```
might be better here? (from `openpype.pipeline.load.LoadError`)